Accessories/ExpansionBoard_uInverter/REV20220329B/uInv_revB_BOM.csv

```diff
@@ -0,0 +1,21 @@
+Supplier Part Number 1,Quantity,Comment,Description,Designator,Footprint,LibRef
+
+"399-17618-1-ND","3","0.1uF","Cap Ceramic 0.1uF 35V X7R 10% SMD 0805","C1A, C1B, C1C","0805_CAP","C0805C104K6RACTU"
+"490-10514-2-ND","5","10uF","0805 35 V 10 uF �10% Tolerance X5R Multilayer Ceramic Capacitor","C2A, C2B, C2C, C5, C6","0805_CAP","GRM21BR6YA106KE43L"
+"445-7551-1-ND","6","2.2uF","2.2 �F �10% 35V Ceramic Capacitor X7R 0805","C3A, C3B, C3C, C4A, C4B, C4C","0805_CAP","C2012X7R1V225K125AB"
+"P19719CT-ND","2","1000u","Cap Aluminum 1000uF 35V 20% (12.5 X 25mm) Radial 5mm 3190mA 7000 hr 105C Ammo Pack","C7, C8","CAP_PAN_12.5X25_PAN","EEU-FM1V102B"
+"311-1142-1-ND","1","0.1uF","Capacitor 0.1uF 16V 0805 SMD X7R","C9","0805_CAP","CC0805KRX7R7BB104"
+"CP-037AH-ND","1","PJ-037AH","CONN PWR JACK 2X5.5MM SOLDER","CONN1","CUI_PJ-037AH","PJ-037AH"
+"609-2802-ND","1","ICD15S13E6GV00LF","Conn High Density D-Sub SKT 15 POS 2.28mm Solder RA Thru-Hole 15 Terminal 1 Port","CONN2","ICD15S13E6GV00LF","ICD15S13E6GV00LF"
+"RJSAE-5385-02-ND","1","RJSAE-5385-02","CONN MOD JACK 8P8C R/A SHIELDED","CONN3","CONN_RJSAE538502_AMP","RJSAE-5385-02"
+"HS404-ND","3","529702B02500G","Heat Sink Passive TO-220 Radial Thru-Hole 2.6C/W Black Anodized","HS1, HS2, HS3","530002","529702B02500G"
+"277-1667-ND","1","1935161","Conn Terminal Blocks 2 POS 5mm Solder ST Thru-Hole 16A","J1","PHOENIX_1935161","1935161"
+"553-3011-1-ND","3","100u","FIXED IND 100UH 5.3A 118 MOHM","L1A, L1B, L1C","FP-PA4344_104NLT-MFG","PA4344.104NLT"
+"36-5005-ND","15","Test Point","Conn Test Point SKT 1 POS ST Thru-Hole 1 Port","PWM-A, PWM-B, PWM-C, TP1A, TP1B, TP1C, TP2A, TP2B, TP2C, TP3A, TP3B, TP3C, TP4, TP5, TP6","TP_1.6MM","Test Point 36-5005ND"
+"A97581-ND","3","5-1634503-1","CONN BNC JACK STR 50 OHM PCB","Q1A, Q1B, Q1C","5-1634503-1","5-1634503-1"
+"541-1.00CCCT-ND","3","1R","Res Thick Film 0805 1 Ohm 1% 1/8W","R1A, R1B, R1C","0805_RES","CRCW08051R00FKEA"
+"541-2841-1-ND","2","2.2K","Res Thick Film 0805 2.2K Ohm 1% 0.4W","R2, R3","0805_RES","RCS08052K20FKEA"
+"RHM1.10KAECT-ND","2","1.1K","Res Thick Film 0805 1.1K Ohm 1% 0.4W","R4, R5","0805_RES","ESR10EZPF1101"
+"15FR150E-ND","3","150m","Resistor; Wirewound; Res 0.15 Ohms; Pwr-Rtg 5 W; Tol 1%; Axial; Silicone; Cur-Sense","Rsense1A, Rsense1B, Rsense1C","RES_15_OHM","15FR150E"
+"IXDN614CI-ND","3","IXDN614CI","14-Ampere Low-Side Ultrafast MOSFET Driver 5-Lead TO-220","U1A, U1B, U1C","IXYS_TO-220-5_IXY","IXDN614CI"
+"INA143U-ND","3","INA143U","SP Amp Difference Amplifier Single �18V/36V 8-Pin SOIC Tube","U2A, U2B, U2C","SOIC127P599X175-8N","INA143U"
```

Comment:
Can you remove the "funny" text characters from the BOM?

![image](https://user-images.githubusercontent.com/20168990/161800486-82f4b559-0057-4da7-b5e4-73f10dd8c6c1.png)

Reply:
Removed.